internal/controller/streams.go

```diff
@@ -449,20 +449,38 @@ func ConnectStreams(ctx context.Context, logger *zap.SugaredLogger, envMap Envir
 			}
 		}
 	}()
-	// Timer channel for 5 seconds
-	resetTimer := time.NewTimer(time.Second * 5)
+
+	// We want to avoid many cloud-operator instances all trying to authenticate
+	// at the same time.
+	//
+	// To that effect, we add a 5 second sleep with 20% jitter here. That way even
+	// if you onboard 100 cloud-operators at the same time, they won't all be
+	// synced up. Normal network delays may do this for us, but we don't want to
+	// rely on that.
+	resetTimer := time.NewTimer(jitterTime(5*time.Second, 0.20))
+	attempt := 0
+
+	// The happy path blocks inside the for loop.
+	// The unhappy path exits the for loop and hits the top-level select.
 	for {
+		failureReason := "We have not failed"
+		attempt++
+		logger.Infow("Trying to authenticate and open streams", "attempt", attempt)
+
 		select {
 		case <-ctx.Done():
+			logger.Info("Context canceled while we were trying to authenticate and open streams")
```

Comment:
1. Remove "we".
1. Move to warn level?
```suggestion
			logger.Warn("Context canceled while trying to authenticate and open streams")
```